code/style.css

```diff
@@ -1,3 +1,84 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #F7D4BC;
+  font-family: 'Gemunu Libre', sans-serif;
+}
+
+header {
+  text-align: center;
+  font-size: 30px;
+}
+
+.profile {
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  text-align:center;
+  color: #218380;
+  margin:auto;
+  width: 100%;
+  flex-direction: column;  
+}
+
+img {
+  border-radius: 50%;
+  width: 200px;
+  margin: 15px;
+  box-shadow: -30px 30px 30px rgba(0, 0, 0, 0.3);
+}
+
+p {
+  color: #AD5D4E;
+}
+
+.projects {
+  flex-direction: column;
```

Comment:
I like your responsiveness of the project-boxes! Since it seems like you have already declared the .projectbox to be display: flex I think that the display: block doesn't do anything in this part? I tried to remove it in the inspector and it seems to no change anything. just a suggestion of you want to have a little bit less code 😊